app/views/support/_time_constraint.html.erb

```diff
@@ -24,10 +24,15 @@
 
     <div class="form-group">
       <span class="form-label">
-        <%= r.label :time_constraint_reason, "Reason for the above dates" %>
+        <%= r.label :time_constraint_reason, "Reason for deadline" %>
       </span>
+      <div class="help-block hint-block--md-6" id="reason-for-deadline-hint">
+        <p>For example, policy or campaign launch, start of the new tax year.</p>
+      </div>
       <span class="form-wrapper">
-        <%= r.text_area :time_constraint_reason, required: false, class: "input-md-6 form-control", rows: 6, cols: 50 %>
+        <%= r.text_area :time_constraint_reason, required: false, class: "input-md-6 form-control", rows: 6, cols: 50, aria: {
+          describedby: "reason-for-deadline-hint"
+        } %>
```

Comment:
Finally, it looks like you still need a 'legal deadline' field:

```
Is this a legal deadline?
[RADIO BUTTON FIELD]
```

This can be added in the same way as the other fields you've already added, as it doesn't factor into the 'time_constraint_attributes' at all.